app/boards/shields/architeuthis_dux/architeuthis_dux.keymap

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+
+	keymap {
+		compatible = "zmk,keymap";
+
+		// This is a sample keymap intended to be replaced with your own
+		base_layer {
+			bindings = <
+				&kp Q     &kp W     &kp E     &kp R     &kp T     &kp Y     &kp U     &kp I     &kp O     &kp P
+				&kp A     &kp S     &kp D     &kp F     &kp G     &kp H     &kp J     &kp K     &kp L     &kp SEMI
+				&kp Z     &kp X     &kp C     &kp V     &kp B     &kp N     &kp M     &kp COMMA &kp DOT   &kp SLASH
+				                              &kp TAB   &kp BSPC  &kp SPACE &kp ENTER
+			>;
+		};
+
+	};
```

Comment:
I've included this simple keymap comprised of only a QWERTY layer, because Architeuthis Dux doesn't have an official keymap. I am happy to put together a more featureful keymap if necessary.

Reply:
I don't think this is a blocker for merging this, especially if upstream doesn't have a "stock" keymap. 

At some point, it might be nice for us to have a standard "default layout" for various stndard-ish layouts, like 36- and 34-key splits, that could be `#include`'d for this exact use case, but this is fine for now.